pkg/cmd/cli/describe/describer.go

```diff
@@ -12,6 +12,7 @@ import (
 	kctl "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
 
+	"encoding/json"
```

Comment:
Definitely not an issue but do you mind grouping imports like (stdlib, 3rd party, openshift) ? It's more convenient when you iterate over them.